%YAML:1.0
---
calibration_time: "Thu 06 Jun 2019 04:48:38 AM EDT"
image_width: 640
image_height: 480
flags: 0
camera_matrix: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 6.1023208536801644e+02, 0., 2.8100271474190458e+02, 0.,
       6.0703459577825038e+02, 2.3133282125557989e+02, 0., 0., 1. ]
distortion_coefficients: !!opencv-matrix
   rows: 1
   cols: 5
   dt: d
   data: [ 1.7824188190520332e-01, -1.5436819158652828e+00,
       2.1529818237688769e-05, -1.0482171380343384e-02,
       4.0015707330249564e+00 ]
avg_reprojection_error: 2.7704090975421689e+00
